src/shim/telemetryClient.ts

```diff
@@ -191,7 +194,6 @@ export class TelemetryClient {
         };
         if (telemetry.dependencyTypeName) {
             if (telemetry.dependencyTypeName.toLowerCase().indexOf("http") > -1) {
-                attributes[SemanticAttributes.HTTP_METHOD] = "HTTP"; // TODO: Dependency doesn't expose method in any property
```

Comment:
This setting is not exposed in telemetry, so we can remove it.

Reply:
If you remove this, how can Exporter know the type?

Reply:
Ah, my mistake. Tested dependency auto-collection which wasn't affected by this change. Looks like manual dependency telemetry registers the type properly and sets `HTTP` correctly.

Reply:
We allow customers to send dependency Type before, we should keep support for that, that is what the TODO was there for, not all dependencies are HTTP

Reply:
Created task to track this and investigate what values we need to support for dependency type.